### DISCUSSION TITLE
Server api settings

### DISCUSSION THREAD

**Discussion by Cchumi** [rustdesk/rustdesk#1832] [Q&A]
Hi a quick question. 

1/ What the fundamentals of the server api and key settings ? 

2/ Is it implemented to be used in RustDesk client ? 

3/ Could it be useful for an address book approach ?

thanks for your answer and advice. 

Cchumi

**Comment by stamepicmorg** [comment]
From FAQ about api and adressbook:
`Change id, addressbook/api/login and web console?`
https://github.com/rustdesk/rustdesk-server/issues/22
https://github.com/rustdesk/rustdesk/discussions/570
https://github.com/rustdesk/rustdesk/issues/509

About key:
https://rustdesk.com/docs/en/self-host/install/#key
